addons/mail/models/mail_template.py

```diff
@@ -623,7 +623,9 @@ def send_mail(self, res_id, force_send=False, raise_exception=False, email_value
             attachment_ids.append((4, Attachment.create(attachment_data).id))
         if attachment_ids:
             mail.write({'attachment_ids': attachment_ids})
-
-        if force_send:
-            mail.send(raise_exception=raise_exception)
+        try:
+            if force_send:
+                mail.send(raise_exception=raise_exception)
+        except:
```

Comment:
I meant like this
```suggestion
        except MailDeliveryException as mde:
             if len(mde.args) == 2 and isInstance(mde.args[1], ConnectionError):
```

Reply:
> I meant like this

Hello @thiryr 
changes have been done, kindly review them.
Thanks.